internal/pkg/pkg.go

```diff
@@ -536,3 +536,28 @@ func FunctionConfigFilePaths(rootPath types.UniquePath, recursive bool) (sets.St
 	}
 	return fnConfigPaths, nil
 }
+
+// FunctionConfigFilterFunc returns a kio.LocalPackageSkipFileFunc filter which will be
+// invoked by kio.LocalPackageReader when it reads the package. The filter will return
+// true if the file should be skipped during reading. Skipped files will not be included
+// in all steps following.
+func FunctionConfigFilterFunc(pkgPath types.UniquePath, includeMetaResources bool) (kio.LocalPackageSkipFileFunc, error) {
+	if includeMetaResources {
+		return func(relPath string) bool {
+			return false
+		}, nil
+	}
+
+	fnConfigPaths, err := FunctionConfigFilePaths(pkgPath, true)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get config paths for the functions in pipeline: %w", err)
```

Comment:
Same as before. "config paths" won't mean much to the user. Are you referring to values specified in `configPath`? It's a specific field name with a proper noun.

Reply:
Updated to directly return error since wrapping it here looks not very useful.